addons/website/static/src/js/widgets/theme.js

```diff
@@ -271,32 +255,18 @@ var ThemeCustomizeDialog = Dialog.extend({
         }
 
         $contents.remove();
-        this.$el.append(core.qweb.render('website.theme_customize_modal_layout'));
-        var $navLinksContainer = this.$('.nav');
-        var $navContents = this.$('.tab-content');
+        this.setElement(core.qweb.render('website.theme_customize_menu_layout'));
```

Comment:
Instead of using `website.theme_customize` as template and doing a `setElement` here could we do the opposite ?
Using `website.theme_customize_menu_layout` as template and simply analyzing the content of `website.theme_customize` here without the need to do a `setElement` ? Feels more logical and should not change a lot of things.